src/librustdoc/clean/utils.rs

```diff
@@ -453,3 +453,18 @@ crate fn find_nearest_parent_module(tcx: TyCtxt<'_>, def_id: DefId) -> Option<De
         None
     }
 }
+
+/// Checks that one attribute is `doc`. For example:
+///
+/// ```text
+/// #[doc(spotlight)]
+/// ```
+///
+/// This function has to exists because it runs on `hir::Attributes` whereas the other runs on
```

Comment:
nit:

```suggestion
/// This function has to exist because it runs on `hir::Attributes` whereas the other runs on
```

or

```suggestion
/// This function exists because it runs on `hir::Attributes` whereas the other runs on
```

Reply:
Might also be worth mentioning what the other function is, but not a big deal.

Reply:
Did it. :)